rules/order.js

```diff
@@ -1,9 +1,12 @@
+// Order inspired by Concentric CSS
+// http://rhodesmill.org/brandon/2011/concentric-css/
 module.exports = {
   'order/properties-order': [
     {
       order: 'strict',
       properties: [
         'content',
+        'quotes',
       ],
     },
     {
```

Comment:
I would actually expect `z-index` to immediately follow a `position` declaration. Sure there are several properties that create a stacking context - but `position` is the main one we all think of.

I'm not super opinionated about this tho.